tutorials/fluidSolidInteraction/beamInCrossFlow/Allrun

```diff
@@ -29,10 +29,17 @@ fi
 # Create plots
 if command -v gnuplot &> /dev/null
 then
+
     echo "Generating deflection.pdf using gnuplot"
-    gnuplot deflection.gnuplot &> /dev/null
+    gnuplot deflection.gnuplot &> /dev/null  
```

Comment:
Thanks Amirhossein!

The trailing whitespace should be removed here. This can be done manually or, for example, in emacs with `M-z whitespace-cleanup`.

Reply:
Resolved in b0767654